.gitignore

```diff
@@ -20,3 +20,4 @@ rSFSW2.Rcheck
 src/*.o
 src/*.so
 src/*.dll
+.Rproj.user
```

Comment:
Please remove this line of code. I think it is best just not to _ever_ put a Rproj in Git folder. All of that development (integration tests, etc.) should take place in a non-git location.

Reply:
I like the idea of having this in the `.gitignore` and would vote to keep it. It doesn't bother here and prevents any potential issues with people creating a Rstudio project inside their git clone of rSWSF2 and attempting to push it up to remote.